app/dashboard/templates/profiles/profile.html

```diff
@@ -7,8 +7,16 @@
     {% endif %}
     {% include 'shared/head.html' %}
     {% include 'shared/cards_pic.html' %}
-    <link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
+	<link rel="stylesheet" href={% static "v2/css/dashboard.css" %} />
+	{% if is_my_profile and theme_type == 'light' %}
```

Comment:
you can remove the `is_my_profile` from these conditionals..  peoples theme colors should show up for everyone!!